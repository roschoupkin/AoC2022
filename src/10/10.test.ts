import * as path from 'path';

import { one } from './solution.1';
import { two } from './solution.2';

describe('10', () => {
  test('part one', () => {
    expect(one(path.join(__dirname, 'puzzle', 'example.txt'))).toBe(13140);
    expect(one(path.join(__dirname, 'puzzle', 'puzzle.txt'))).toBe(14620);
  });

  test('part two', () => {
    expect(two(path.join(__dirname, 'puzzle', 'example.txt'))).toMatchSnapshot();
    expect(two(path.join(__dirname, 'puzzle', 'puzzle.txt'))).toMatchSnapshot();
  });
});
