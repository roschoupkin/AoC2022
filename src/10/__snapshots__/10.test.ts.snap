// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`10 part two 1`] = `
"
##..##..##..##..##..##..##..##..##..##..
###...###...###...###...###...###...###.
####....####....####....####....####....
#####.....#####.....#####.....#####.....
######......######......######......####
#######.......#######.......#######.....
"
`;

exports[`10 part two 2`] = `
"
###....##.####.###..#..#.###..####.#..#.
#..#....#.#....#..#.#..#.#..#.#....#..#.
###.....#.###..#..#.####.#..#.###..#..#.
#..#....#.#....###..#..#.###..#....#..#.
#..#.#..#.#....#.#..#..#.#.#..#....#..#.
###...##..#....#..#.#..#.#..#.#.....##..
"
`;
